Code/Core/FileUtils.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include <string>
+
+namespace core
+{
+	class FileUtils
+	{
+	public:
+		static std::string ReadFile(std::string path);
+		static void CreateNewFile(std::string path, std::string content);
+		static void OverWriteExistingFile(std::string path, std::string content);
+		static void AppendToExistingFile(std::string path, std::string content);
+		static std::string GetProjectDirectory();
+		static std::string GetDataDirectory();
+		static std::string ConvertOSPathString(std::string path);
```

Comment:
Can we pass strings by const reference to these functions?

- Avoids calling the copy constructor on the string
- Makes the reference immutable (since most of these are paths or are shifted into output streams and we don't modify them)
```
const std::string& path 
```